pkg/boot/boot.go

```diff
@@ -0,0 +1,22 @@
+package boot
+
+import (
+	"log"
+)
+
+// OSImage represents a bootable OS package.
+type OSImage interface {
+	ExecutionInfo(log *log.Logger)
+
+	// Execute kexec's the OS image: it loads the OS image into memory and
+	// jumps to the kernel's entry point.
+	//
+	// If dryRun is true, Execute should write the OS image to tmpfs and
```

Comment:
dryRun is no more, but this comment was not updated

Reply:
Done.